src/components/tiger/tiger.tsx

```diff
@@ -0,0 +1,252 @@
+        setHasSaved(true); // Set hasSaved state to true
+        setAllowSave(false); // Disallow further output saving
+    };
+
+    /**
+     * Clears the command output and resets state variables related to output saving.
+     */
+    const clearOutput = () => {
+        setOutput(""); // Clear the command output
+        setHasSaved(false); // Reset hasSaved state
+        setAllowSave(false); // Disallow further output saving
+    };
+
+    // Render component
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                />
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButton(loading, pid)}
+                    <TextInput
+                        label="Target IP/Hostname"
+                        required
+                        {...form.getInputProps("targetIP")}
+                        placeholder="e.g. 192.168.1.1"
+                    />
+                    <TextInput
+                        label="Audit Level"
+                        required
+                        {...form.getInputProps("auditLevel")}
+                        placeholder="e.g. 5"
+                    />
+                    <TextInput
+                        label="Report File"
+                        {...form.getInputProps("reportFile")}
+                        placeholder="e.g. /path/to/report.txt"
+                    />
+                    <TextInput
+                        label="Enable Modules"
+                        {...form.getInputProps("enableModules")}
+                        placeholder="e.g. passwd, fs"
+                    />
+                    <TextInput
+                        label="Exclude Modules"
+                        {...form.getInputProps("excludeModules")}
+                        placeholder="e.g. account, cron"
+                    />
+                    <Checkbox
+                        label="Verbose Mode"
+                        checked={verboseMode}
+                        onChange={(event) => setVerboseMode(event.currentTarget.checked)}
+                    />
+                    <Button type="submit" disabled={loading}>
+                        Run Audit
+                    </Button>
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
+                    <Button onClick={clearOutput} disabled={loading || output.length === 0}>
+                        Clear Output
+                    </Button>
+                </Stack>
```

Comment:
This is the way the clear output button should be formatted. 
```suggestion
  <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
                </Stack>
```